content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,300 @@
+values. When a tracked param is changed, `dvc exp run` invalidates any stages
+that depend on it, and reruns the experiment.
+
+> Parameters can be defined in `dvc.yaml` directly or through `dvc stage add`.  
+> 📖 See `dvc params` for more details.
+
+For a params file named `params.yaml` with the contents
+
+```yaml
+model:
+  learning_rate: 0.0001
+```
+
+You can specify the parameter dependency as
+
+```dvc
+$ dvc stage add -n train \
+                --parameter model.learning_rate \
+                --outs ...
+```
+
+> ⚠️ DVC does not check whether the parameters are actually used in your code.
+
+<details>
+
+#### Non-default parameter files
+
+DVC allows param files in YAML 1.2, JSON, TOML, and Python formats. When your
+parameters file is named something other than `params.yaml`, you need to specify
+it in both stage description and `dvc exp run`. For example using
+`myparams.toml`:
+
+```dvc
+$ dvc stage add -n train \
+                -p myparams.toml:learning_rate \
+                ...
+
+$ dvc exp run -S myparams.toml:learning_rate = 0.0001
+```
+
+</details>
+
+### Updating experiment parameters on-the-fly
+
+DVC allows to update the parameters from command line when running
+`dvc experiments`. The `--set-param` (`-S`) option takes a parameter name and
+its value, and updates the params file before the run.
+
+```dvc
+$ dvc exp run --set-param model.learning_rate=0.0002
+```
+
+> Note that parameters are attached to experiments so you can view them together
+> with `dvc exp show` and `dvc exp diff`.
+
+To set more than one param for the same experiment, use the `-S` option multiple
+times, or supply a comma-delimited list to `-S`:
+
+```dvc
+$ dvc exp run -S learning_rate=0.001 -S units=128
+# is equivalent to
+$ dvc exp run -S learning_rate=0.001,units=128
+```
+
+> ⚠️ Note that DVC doesn't check whether parameters given to `--set-param` are
+> already in the parameters file. If there is a typo, a new or different param
+> will be added/changed.
+
+## The experiments queue
+
+The `--queue` option of `dvc exp run` tells DVC to append an experiment for
+later execution. Nothing is actually run yet.
+
+<details>
+
+### How are experiments queued?
+
+Queued experiments are created similar to
+[Git stash](https://www.git-scm.com/docs/git-stash). The last experiment queued
+is found in `.git/refs/exps`, and earlier ones are in its [reflog].
+
+[reflog]:
+  https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefreflogareflog
+
+</details>
+
+```dvc
+$ dvc exp run --queue -S units=10
+Queued experiment '1cac8ca' for future execution.
+$ dvc exp run --queue -S units=64
+Queued experiment '23660bb' for future execution.
+$ dvc exp run --queue -S units=128
+Queued experiment '3591a5c' for future execution.
+$ dvc exp run --queue -S units=256
+Queued experiment '4109ead' for future execution.
+```
+
+> `dvc exp show` will mark queued experiments with an asterisk `*`.
```

Comment:
This is no longer true, and it's probably best to stay away from trying to describe the output, especially for experiments, since it's not really stable yet. The table now has a column that lists "Queued" for the relevant experiments, which hopefully makes it self-evident.

Reply:
Removed.